drivers/integration/docker/docker.go

```diff
@@ -286,13 +353,18 @@ func (d *driver) fsType() string {
 	return d.config.GetString("docker.fsType")
 }
 
+func (d *driver) mountDirPath() string {
+	return d.config.GetString("docker.mountDirPath")
+}
+
 func configRegistration() *gofig.Registration {
 	r := gofig.NewRegistration("Docker")
 	r.Key(gofig.String, "", "ext4", "", "docker.fsType")
 	r.Key(gofig.String, "", "", "", "docker.volumeType")
 	r.Key(gofig.String, "", "", "", "docker.iops")
 	r.Key(gofig.String, "", "", "", "docker.size")
 	r.Key(gofig.String, "", "", "", "docker.availabilityZone")
+	r.Key(gofig.String, "", "/var/lib/rexray/volumes", "", "docker.mountDirPath")
```

Comment:
We should set this value in the config instance from REX-Ray and not to this path inside libStorage. Don't you think?